Plugins/UnityClientPlugin/MediaEngineUWP/Shared/MediaEnginePlayer.cpp

```diff
@@ -112,7 +112,9 @@ MEPlayer::MEPlayer(Microsoft::WRL::ComPtr<ID3D11Device> unityD3DDevice) :
 	m_d3dFormat(DXGI_FORMAT_B8G8R8A8_UNORM),
 	m_fInitSuccess(FALSE),
 	m_fExitApp(FALSE),
-	m_fUseDX(TRUE)
+	m_fUseDX(TRUE),
+	m_fUseVSyncTimer(useVSyncTimer),
```

Comment:
nit: what is the significance of the `f` in this naming scheme? and should `m_renderFps` have it?

Reply:
f means flag. It took me a while to understand about the naming convention used in media engine player. :)